system/keymaps/keyboard.xml

```diff
@@ -48,7 +48,7 @@
       <browser_back mod="longpress">ActivateWindow(Home)</browser_back>
       <key id='65446'>Back</key>
       <m>Menu</m>
-      <m mod="ctrl">ActivateWindow(PlayerControls)</m>
+      <n mod="ctrl">ActivateWindow(PlayerControls)</m>
```

Comment:
AS @Razzeee said the closing tag needs also be changed to `</n>` because it's  `</m>` atm.